MANIFEST.in

```diff
@@ -0,0 +1,37 @@
+recursive-include unionml/templates *
+
+include README.md
+include src/pythae/py.typed  # marker file for PEP 561
+
+include CHANGELOG.md
+include changelog.md
+include LICENSE
+include LICENSE.txt
+include CITATION.cff
+include *.cff  # citation info
+
+include MANIFEST.in
+include src/pythae/pyproject.toml
+include setup.py
+include setup.cfg
+include tox.ini
```

Comment:
No such file in repo

Reply:
This was copied from a standard template of mine. Yes, in this case it can be dropped.

Reply:
Done